manager/orchestrator/task_reaper.go

```diff
@@ -75,13 +79,30 @@ func (tr *TaskReaper) Run() {
 				}] = struct{}{}
 				if len(tr.dirty) > maxDirty {
 					tr.tick()
+					if timer != nil {
+						timer.Stop()
+					}
+					timerCh = nil
```

Comment:
According to code, it doesn't change channel for the timer. https://golang.org/src/time/sleep.go?s=3155:3193#L95

It also recommends the doing timer.Stop and clear channel before timer.Reset. 
```
    94	// 	if !t.Stop() {
    95	// 		<-t.C
    96	// 	}
    97	// 	t.Reset(d)
```